app/routes/events/components/EventDetail/index.js

```diff
@@ -129,6 +130,20 @@ export default class EventDetail extends Component<Props> {
       ? () => unfollow(event.isUserFollowing.id, event.id)
       : () => follow(currentUser.id, event.id);
 
+    const formatTime = (time: moment) => {
+      const dateTime = moment(time);
```

Comment:
You dont need to moment(time), time is already a moment object :) startTime, endTime are moment already as well

Reply:
That's what I thought too but that wasn't the case for `event.paymentDueDate` and some other the others

Reply:
Then fix the transform for those dates in the same way as startTime instead :)

Reply:
So I guess it's incorrect flow to say `(time: moment)` but my code crashed when assuming the parameter was already a `moment`.